src/binary/ibc-setup/commands/init.ts

```diff
@@ -57,6 +97,71 @@ export async function run(options: Options) {
   }
 
   const registry = yaml.load(fs.readFileSync(registryFilePath, 'utf-8'));
-  // TODO #75: registry file validation
-  console.log(registry);
+
+  const ajv = new Ajv({ allErrors: true });
+  const schema: JSONSchemaType<Registry> = {
```

Comment:
Nice one.
I'd love to put this in validation logic, and maybe load load and validate, in another file (but as you said, that can be next PR)

Reply:
Yes, especially that some functionality must be shared across the commands. I think it's better to start working on this while having at least 2 usages.